### ISSUE TITLE
CUSA05670 - WIPEOUT™ OMEGA COLLECTION

### ISSUE THREAD

**Issue by Crafty-The-Fox** [shadps4-emu/shadps4-game-compatibility#152]
### Game Name

WIPEOUT™ OMEGA COLLECTION

### Game code

CUSA05670

### Game version

v1.00

### Used emulator's version (only released versions are acceptable)

v0.1.0

### Current status

Nothing

### Error

hangs after this, cursor still blinking though. game probably dosent know what to do with the responses it got from stubs.

```
[Kernel.Pthread] <Info> thread_management.cpp:scePthreadCondWait:1066: scePthreadCondWait, result=0
[Kernel.Fs] <Info> file_system.cpp:sceKernelClose:124: Closing /app0/data00.psarc
```

### Description

n/a

### Log File

[shad_log.txt](https://github.com/user-attachments/files/16154962/shad_log.txt)

**Comment by Crafty-The-Fox** [comment]
v0.2.0 seemingly crashes in an unvalid texture type

```
[Debug] <Critical> vk_pipeline_cache.cpp:CreateGraphicsPipeline:296: Unreachable code!
Invalid texture type Color2DMsaa
```
[shad.log](https://github.com/user-attachments/files/16672092/shad.log)

**Comment by Crafty-The-Fox** [comment]
v0.4.0 crashes with:
```
[Debug] <Critical> image_info.cpp:operator():277: Assertion Failed!
```
[shad_log.txt](https://github.com/user-attachments/files/17697477/shad_log.txt)

**Comment by jaloji** [comment]
Game in v1.07 and shadPS4 v0.4.0, nothing and crash with:
`[Debug] <Critical> address_space.cpp:operator():187: Assertion Failed!
(FormatMessageA failed to format error)`

[shad_log.txt](https://github.com/user-attachments/files/17822450/shad_log.txt)

**Comment by Savemypeanuts** [comment]
Bump. Please fix, still crashes but it would be easy to fix for the right person. Please may you if you're that genius solve this issue and get this game running, you will become blessed if so. I guarantee that.

**Comment by Crispy81** [comment]
0.4.1 WIP main 6c215e6 boots to the splash screens and autosave info screen, then crashes and closes.

![Screenshot 2024-11-30 145957](https://github.com/user-attachments/assets/9ad56062-28e2-4844-8541-478f571ba209)
![Screenshot 2024-11-30 150005](https://github.com/user-attachments/assets/b7c4535c-982e-410e-bb89-cadeb0744644)
![Screenshot 2024-11-30 150012](https://github.com/user-attachments/assets/4cb6ffa8-bd62-4468-a278-a7245fb19afa)

[shad_log.txt](https://github.com/user-attachments/files/17964133/shad_log.txt)

**Comment by Crispy81** [comment]
What OS is that one? For me, on Windows, I can't get that far on the latest build.

**Comment by tomboylover93** [comment]
It's Linux (with the Qt AppImage build). I don't have a Windows install and I was hoping someone could test the Windows build for that commit and send logs so I'd know if it's necessary to open a separate issue for Linux.

**Comment by Crispy81** [comment]
I'm using latest build on Windows and it crashes at the same place as my previous post with screenshots

[shad_log.txt](https://github.com/user-attachments/files/17984037/shad_log.txt)

**Comment by tomboylover93** [comment]
Just to check, do you happen to have the firmware modules in place? I forgot to do that and now that I have them the game goes so far as to getting into a race, but the only thing that renders at all is the HUD. I'll have to open a separate issue for Linux but the 0.4.0 release build gives me a segmentation fault and refuses to launch so I'll have to figure that out.

**Comment by Crispy81** [comment]
Yes, got them in place.

![image](https://github.com/user-attachments/assets/0c1999bc-dc5f-4051-971f-feb185453bda)

**Comment by tomboylover93** [comment]
I opened [an issue for Linux](https://github.com/shadps4-emu/shadps4-game-compatibility/issues/1326), and I also noticed that my game code is different (CUSA07671 is for the US version whereas CUSA05670 is for the European version. Maybe there are differences in playability between regional versions? IDK). If someone has version CUSA07671 of the base (1.00) game and is on Windows (or even better, Linux with an NVIDIA/Intel graphics card) I'd appreciate it if you shared any logs or screenshots.

**Comment by tomboylover93** [comment]
@Crispy81 Might be a stretch but on the log file you provided for 0.4.1 WIP main fda4f06 your game crashes right after `[Lib.Ajm] <Info> ajm.cpp:sceAjmInstanceCreate:137: called context = 1, codec_type = At9Dec, flags = 0x11`, and from looking at the source code Ajm has to do with audio playback and after that last screenshot you showed it's supposed to get to the main menu and start playing music. Try creating a folder named CUSA05670 under sys_modules (if you haven't already) and then placing libSceAjm.sprx there.

**Comment by Crispy81** [comment]
Tried that, same result.

**Comment by tomboylover93** [comment]
I see. You could probably try your luck with the various forks until a developer picks this up as I've seen games working differently with forks designed for Bloodborne, which implement stuff like tesselation. I myself have gotten ingame with the latest commit and the issues I've experienced are related to textures and shaders so I'll leave it at that until someone who knows Vulkan looks into it.

**Comment by Crafty-The-Fox** [comment]
i have no clue what is going on, HOWEVER on v0.5.0 it still shwos nothing and crashes before anything is displayed.

[shad_log.txt](https://github.com/user-attachments/files/18579765/shad_log.txt)

**Comment by Savemypeanuts** [comment]
Yup, we need to get this game working within 2 years!!! I have the PC for
it, just need a super nerd. I will pay you $200 if you get it working! I'm
rich

**Comment by Azrael-3** [comment]
I'm on v 0.6.0 
win 10 ,rtx3060, i5 12700k
game crashes before even booting

[shad_log.txt](https://github.com/user-attachments/files/18619375/shad_log.txt)

**Comment by tomboylover93** [comment]
I've tested it on Windows with 0.6.0 and it goes ingame. I have a Radeon RX 5500XT, so the crashing other people reported might be an NVIDIA-only issue.

![Image](https://github.com/user-attachments/assets/ddb100a0-52b6-479f-97b1-8d13cde15e86)

![Image](https://github.com/user-attachments/assets/6436c10b-58e9-4fea-a5f1-f7810dcc2cfc)

[shad_log.txt](https://github.com/user-attachments/files/18756873/shad_log.txt)

**Comment by Savemypeanuts** [comment]
For real? You get gameplay? I've got a 7900xtx. Could maybe get a lap if
that's true, tops lol

**Comment by Crafty-The-Fox** [comment]
Crashes with nvidia on a 3080 and a 9900k on windows 10

**Comment by tomboylover93** [comment]
Yeah, it crashes with NVIDIA cards. I was using AMD. I'm pretty sure that it's a device lost error, so you should install the Vulkan SDK and enable Crash Diagnostics and Vulkan Validation Layers to generate a cdl_dump.yaml file that the devs can use to debug it.

**Comment by smiRaphi** [comment]
same error with game version 1.07 on a 4090 & 7950x3D

[shad_log.txt](https://github.com/user-attachments/files/18970200/shad_log.txt)

**Comment by MaixGit** [comment]
Can confirm, vk_scheduler seems to be an NVIDIA issue most likely. This is with all the sys_modules, also the same error for game v1.0, and the latest nightly build of shadPS4.

I'm not sure if I ran the Vulkan SDK with the correct settings but here are all the logs:

CUSA07671 (USA) v1.07 with a GTX 1080 and 7600x:
[shad_log.txt](https://github.com/user-attachments/files/19008691/shad_log.txt)
[VulkanSDK_log.txt](https://github.com/user-attachments/files/19009132/VulkanSDK_log.txt)
[cdl_dump.yaml.txt](https://github.com/user-attachments/files/19009138/cdl_dump.yaml.txt)

I'll also check with any other regions I might get my hands on, such as CUSA05670 (EUR), shouldn't make a difference though.

**Comment by Olivinee** [comment]
Instantly crashes on launch
Win11, 2080ti, R5 5600x
[shad_log.txt](https://github.com/user-attachments/files/19240035/shad_log.txt)
